hash/fmtowns_flop.xml

```diff
@@ -1524,6 +1524,20 @@ Zurukamashi Ver 2.0                                           Nikkonren Kikaku
 		</part>
 	</software>
 
+	<software name="mightmgw">
+		<description>Might and Magic - World of Xeen</description>
+		<year>1994</year>
+		<publisher>スタークラフト (Starcraft)</publisher>
+		<info name="alt_title" value="ワールド・オブ・ジーン" />
+		<info name="usage" value="Requires 4 MB RAM"/>
+		<info name="usage" value="This disk combines MM4 and MM5 into a single game. Requires both CDs and a save disk from either game. Boot TownsOS 2.1 and run the icon on drive A to install." />
```

Comment:
You can have only one `info name="usage"` field.

Reply:
Silly mistake on my part, I added the install instructions and I forgot I already had a usage field. It's fixed now.

Reply:
This isn’t actually a requirement.  Software can have multiple instances of the same info type, and there are several that do in MAME right now.  Front-ends have to deal with it.  We can find them with a minimaws query:
```
sqlite> SELECT *
    FROM (SELECT
        softwarelist.shortname || ':' || software.shortname as shortname,
        softwareinfotype.name AS info,
        COUNT(softwareinfo.id) AS cnt
        FROM
            softwareinfo
            LEFT JOIN software ON softwareinfo.software = software.id
            LEFT JOIN softwarelist ON software.softwarelist = softwarelist.id
            LEFT JOIN softwareinfotype ON softwareinfo.infotype = softwareinfotype.id
        GROUP BY
            softwareinfo.software,
            softwareinfo.infotype)
    WHERE cnt > 1;
mz700_cass:lissajou|usage|2
mz700_cass:sinberg|usage|2
ibm5170:mission|alt_title|2
fmtowns_cd:bfmaria|serial|2
cgenie_cass:natomors|author|2
cgenie_cass:bumm|author|3
cgenie_cass:fraggels|author|2
cgenie_cass:glueck|author|2
cgenie_cass:grauen|author|2
cgenie_cass:hdeath|author|2
cgenie_cass:toadman|author|2
cgenie_cass:triton|author|2
gba:fantmarc|alt_title|2
gba:2sadvchuj|alt_title|2
gba:2sadvbatj|alt_title|2
msx1_cass:zond|alt_title|2
gbcolor:zhwsj|alt_title|2
gbcolor:zzqb|alt_title|2
gbcolor:lkrx5|alt_title|2
cpc_cass:academy|usage|2
cpc_cass:capblood|usage|2
```

There are already multiple entries in the `cpc_cass` and `mz700_cass` lists that have multiple `usage` items.